operations/network/dns/PowerDNS_Troubleshooting.md

```diff
@@ -0,0 +1,141 @@
+# PowerDNS troubleshooting
```

Comment:
An entry for this new file should be included in the operations/index.md file

Reply:
Renamed document to Troubleshoot_PowerDNS.md for consistency and added link to operations/index.md